Docs: Updating plugin publishing docs (#60853)
grafana/grafana: docs/sources/developers/plugins/publish-a-plugin.md

- Check that your plugin is ready for review using the [plugin validator](https://github.com/grafana/plugin-validator).
- Read our [6 tips for improving your Grafana plugin before you publish](https://grafana.com/blog/2021/01/21/6-tips-for-improving-your-grafana-plugin-before-you-publish/).

> - Check out [plugin-examples](https://github.com/grafana/grafana-plugin-examples) we provide for best practices on building your plugin.

Reviewer: ```suggestion
- Refer to [plugin-examples](https://github.com/grafana/grafana-plugin-examples) to review best practices for building your plugin.
```

- We're not able to give an estimate at this time, though we're constantly working on improving the time it takes to review a plugin.

**Can I decide a date when my plugin will be published?**

> - No. We can not guarantee specific publishing dates as plugins are being immediately published after a successfull review based on our internal prioritization.

Reviewer: ```suggestion
- No. We cannot guarantee specific publishing dates, as plugins are immediately published after a review based on our internal prioritization.
```
